qiskit/quantum_info/operators/channel/quantum_channel.py

```diff
@@ -232,7 +251,6 @@ def to_instruction(self):
         Raises:
             QiskitError: if input data is not an N-qubit CPTP quantum channel.
         """
-        from qiskit.circuit.instruction import Instruction
```

Comment:
This instruction is necessary, but it may be conflict with `import` at the beginning.

Is the `TYPE_CHECKING` blocking really needed? I don't think `Instruction` is dependent on quantum channel.

Reply:
I believe this is a remnant of a previous measure to circular references. I tried it this time and it seemed to be OK, so I changed it to import `Instructions` in the global scope.